ocs_ci/ocs/node.py

```diff
@@ -85,6 +85,8 @@ def get_nodes(node_type=constants.WORKER_MACHINE, num_of_nodes=None):
         list: The nodes OCP instances
 
     """
+    from ocs_ci.ocs.cluster import is_hci_provider_cluster
```

Comment:
Why are those imports done in a function and not at the start of the file?

Reply:
Because it will cause a circular loop.